lib/puppet/provider/java_ks/keytool.rb

```diff
@@ -54,19 +54,33 @@ def get_password
 
   def password_file
     pword = get_password
+    Puppet.debug("Pword = #{pword.inspect}")
```

Comment:
I obviously wouldn't print the passwords in logs in the final version, as well as most of the other debug statements.